<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>YSYX-IDE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/images.qrc">
    <normaloff>:/images/images/pencil.png</normaloff>:/images/images/pencil.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
  QMenuBar {
      background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                        stop:0 lightgray, stop:1 darkgray);
      spacing: 12px; /* spacing between menu bar items */
	  color:blcak;
  }

  QMenuBar::item {
      padding: 1px 4px;
      background: transparent;
      border-radius: 4px;
  }

  QMenuBar::item:selected { /* when selected using mouse or keyboard */
      background: #a8a8a8;
  }

  QMenuBar::item:pressed {
      background: #888888;
  }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget#listWidget{
	background-color: rgba(255, 255, 255,96);	
    /*border-radius: 40px;*/
	font-size:20px;
}
QListWidget#listWidget::item {
	color: black;
    border-radius: 40px;
    border:none;
    height: 60px;
    text-align: center; /* 设置文本居中 */
}
QListWidget#listWidget::item:selected {
    background-color: rgba(0, 255, 0,99);	
    color:white;
}</string>
      </property>
      <item>
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NVBoard</string>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>100</pointsize>
           </font>
          </property>
          <property name="text">
           <string>NVBOARD</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="recent">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Open Recent</string>
     </property>
    </widget>
    <addaction name="new_file"/>
    <addaction name="open_dir"/>
    <addaction name="recent"/>
    <addaction name="separator"/>
    <addaction name="save_file"/>
    <addaction name="save_as"/>
    <addaction name="separator"/>
    <addaction name="print"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="copy"/>
    <addaction name="paste"/>
    <addaction name="cut"/>
    <addaction name="separator"/>
    <addaction name="font"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Build</string>
    </property>
    <addaction name="about"/>
   </widget>
   <widget class="QMenu" name="menuNvboard">
    <property name="title">
     <string>Nvboard</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>Analyze</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menuDebug"/>
   <addaction name="menuAnalyze"/>
   <addaction name="menuNvboard"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="new_file"/>
   <addaction name="open_dir"/>
   <addaction name="print"/>
   <addaction name="save_file"/>
   <addaction name="save_as"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="copy"/>
   <addaction name="paste"/>
   <addaction name="cut"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="font"/>
  </widget>
  <widget class="QToolBar" name="toolBar_4">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="undo"/>
   <addaction name="redo"/>
  </widget>
  <widget class="QToolBar" name="toolBar_5">
   <property name="windowTitle">
    <string>toolBar_5</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="about"/>
   <addaction name="exit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar_6">
   <property name="windowTitle">
    <string>toolBar_6</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="debug"/>
   <addaction name="actionruntocomp"/>
   <addaction name="actionstep"/>
   <addaction name="actionstepfunc"/>
  </widget>
  <action name="new_file">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/new.png</normaloff>:/images/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="open_dir">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Folder</string>
   </property>
   <property name="toolTip">
    <string>打开文件夹</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="save_file">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="save_as">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/save_as.png</normaloff>:/images/images/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="paste">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/paste.png</normaloff>:/images/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="cut">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/cut.png</normaloff>:/images/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="copy">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/copy.png</normaloff>:/images/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="font">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/font.png</normaloff>:/images/images/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="about">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/info.png</normaloff>:/images/images/info.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="undo">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/edit_undo.png</normaloff>:/images/images/edit_undo.png</iconset>
   </property>
   <property name="text">
    <string>撤回</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redo">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/edit_redo.png</normaloff>:/images/images/edit_redo.png</iconset>
   </property>
   <property name="text">
    <string>取消撤回</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="exit">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="print">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/print.png</normaloff>:/images/images/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="clear_history">
   <property name="text">
    <string>清除打开历史记录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Shift+C</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/info.png</normaloff>:/images/images/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionruntocomp">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/runtocomp.png</normaloff>:/images/images/runtocomp.png</iconset>
   </property>
   <property name="text">
    <string>runtocomp</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionstep">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/step.png</normaloff>:/images/images/step.png</iconset>
   </property>
   <property name="text">
    <string>step</string>
   </property>
  </action>
  <action name="debug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedoff>:/images/images/stop_debug.png</selectedoff>
     <selectedon>:/images/images/start_debug.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>debug</string>
   </property>
  </action>
  <action name="actionstepfunc">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/images/step_func.png</normaloff>:/images/images/step_func.png</iconset>
   </property>
   <property name="text">
    <string>stepfunc</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/images.qrc"/>
 </resources>
 <connections/>
</ui>
